en/rest/schemas.mdown

```diff
@@ -0,0 +1,186 @@
+# Schema
+
+This API allows you to access the Schema of your app.
```

Comment:
I think it should be schema rather than Schema. Lowercase is better cause it's not a proper noun.

Also we need to decide between schema and schemas. I think a single schema is for a single class, and so on this line it would be schemas.